derive/src/pymodule.rs

```diff
@@ -58,10 +58,19 @@ pub fn impl_pymodule(
     // append additional items
     let module_name = context.name.as_str();
     let module_extend_items = context.module_extend_items;
+    let doc = crate::doc::try_read(module_name).ok().flatten();
+    let doc = if let Some(doc) = doc {
+        quote!(Some(#doc))
+    } else {
+        quote!(None)
+    };
```

Comment:
For the previously implemented items, they try to read user defined document first and CPython database document later. Probably because `pymodule` didn't have feature to load user defined document, this is only reading CPython document. I think this behavior is confused for its consistency.

Could you add either user defined document implementation or a `TODO:` comment for prioritized reading user defined document?

Reply:
I made it try to read user-defined documents first in dd5e55d. Please review again 🙏🏻

Reply:
looks good, thank  you!